packages/prismic/README.md

```diff
@@ -0,0 +1,83 @@
+# prismic
+
+> Prismic library for Next
+
+## Installation
+
+With Yarn:
+```bash
+yarn add @vue-storefront/prismic
+```
+
+With NPM:
+```bash
+npm install --save @vue-storefront/prismic
+```
+
+## Setup library
+
+```typescript
+import { setup } from '@vue-storefront/prismic'
+
+setup({
+  endpoint: 'https://your-account.prismic.io/api/v2',
+  apiOptions: {
+    accessToken: 'Prismic_Access_Token'
+  }
+});
+```
+
+## Nuxt support
+
+If you need to integrate Prismic with Nuxt, just simply provide setup file (that one above for example) into `nuxt.config.js` plugins section.
```

Comment:
words like Simply, Just can be demotivating for people struggling with issues on these steps, should remove both and it still will read ok. When suggessting configuration, always provide an example e.g.

---------------------------------------------------------------------------------------

If you need to integrate Prismic with Nuxt, create a setup file (shown in example above) and register it in your `nuxt.config.js` plugins section.

__nuxt.config.js__

```javascript
export default {
  // ...
  plugins: ['./path/to/your-prismic-setup-file.js']
}
```


```